examples/transformers/whisper.py

```diff
@@ -7,8 +8,27 @@
 audio = requests.get("https://www.nasa.gov/62283main_landing.wav").content
 
 task = "automatic-speech-recognition"
+architecture = "openai/whisper-small"
 
-audio_transcription_pipeline = transformers.pipeline(task=task, model="openai/whisper-small")
+model = transformers.WhisperForConditionalGeneration.from_pretrained(architecture)
+tokenizer = transformers.WhisperTokenizer.from_pretrained(architecture)
+feature_extractor = transformers.WhisperFeatureExtractor.from_pretrained(architecture)
+if Version(transformers.__version__) > Version("4.30.2"):
+    model.generation_config.alignment_heads = [
+        [5, 3],
+        [5, 9],
+        [8, 0],
+        [8, 4],
+        [8, 7],
+        [8, 8],
+        [9, 0],
+        [9, 7],
+        [9, 9],
+        [10, 5],
+    ]
```

Comment:
This is an example. We can remove `Version(transformers.__version__) > Version("4.30.2")`.